app/tests/evaluation_tests/test_views.py

```diff
@@ -589,8 +589,12 @@ def test_hidden_phase_visible_for_admins_but_not_participants(client):
         )
         assert response.status_code == status
         if status == 200:
-            assert f"{visible_phase.title}</a>" in str(response.content)
-            assert f"{hidden_phase.title}</a>" not in str(response.content)
+            assert f"</i>&nbsp;&nbsp;{visible_phase.title}" in str(
```

Comment:
This is a bit strange. I think either you test for the icon + title, or you just check for the title. 

The 2 spaces you check for here, by the way, also point me to some improvements in the templates. I will comment there.

Reply:
Indeed, I will update the test